docs/displaysystem/angular.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: page
+title: "Angular Display System"
+description: ""
+group: display
+---
+{% include JB/setup %}
+
+
+### Angular (Beta)
+
+Angular display system treats output as an view template of [AngularJS](https://angularjs.org/).
+It compiles templates and display inside of Zeppelin.
+
+Zeppelin provides gateway between your interpreter and your compiled AngularJS view teamplates.
+Therefore, you can not only update scope variable from your interpreter  but also watch your scope variable in the interpreter, which is JVM process.
+
+<br />
+#### Print AngularJS view
+
+To use angular display system, your output should starts with "%angular".
+<img src="../../assets/themes/zeppelin/img/screenshots/display_angular.png" width=600px />
+
+Note that display system is backend independnet.
+
+Because of variable 'name' is not defined, 'Hello \{\{name\}\}' display 'Hello '.
+
+<br />
+#### Bind/Unbind variable
+
+Through ZeppelinContext, you can bind/unbind variable to AngularJS view.
+
+
+```
+// bind my 'object' as angular scope variable 'name' in current notebook.
```

Comment:
should we indicate the language? or show example in other languages?
for instance, these work in Java and Scala.